### PR TITLE
Disable cloud-init override of NM DNS

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date

### CODE REVIEW

File: ci/playbooks/multinode-customizations.yml [hunk 1]
```diff
@@ -128,6 +128,28 @@
       ansible.builtin.set_fact:
         persistent_ssh_key: "{{ pub_key_slurp['content'] | b64decode }}"
 
+    - name: Check if cloud-init is overriding NM settings
+      become: true
+      ansible.builtin.stat:
+        path: "/etc/NetworkManager/conf.d/99-cloud-init.conf"
+      register: _cloud_init_nm_override_stat
+
```
Comment (cjeanner) [inline ci/playbooks/multinode-customizations.yml:136]: we may want to get handlers for this kind of things. We'd then be able to just get one task, triggering the handler when it changes. But not for now - it's not the only place either.

**Review by rlandy — COMMENTED** [review]
/lgtm

**Pull request merged**